### DISCUSSION TITLE
Twitter returned code 403

### DISCUSSION THREAD

**Discussion by Mozitux** [AzuraCast/AzuraCast#5620] [Q&A]
Hello, I try to use the Twitter webhooks, I've followed the instructions to make an application on Twitter Developer Portal, but when I click on the test button, there is the following error code :

_AzuraCast.DEBUG: Twitter returned code 403 {"response_body":"{\"errors\":[{\"message\":\"You currently have Essential access which includes access to Twitter API v2 endpoints only. If you need access to this endpoint, you’ll need to apply for Elevated access via the Developer Portal. You can learn more here: https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api#v2-access-leve\",\"code\":453}]}\n"} []_

I'm a little lost with what I have to do, and I'm not sure if I have to apply for Elevated access just to be able to making a bot that auto tweet every **x** minutes...

**Comment by pwrj** [comment]
Yes, you have to.
